Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +22,7 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array?.Length > 0) Array.Reverse(array);
```

Comment:
Hah! Please do not cheat, before you can use these functinos, you need to understand what happens under the hood, please implement it yourself.

Reply:
Whoops, busted! Okay, I'll try it the proper way.